client/components/checkout.js

```diff
@@ -54,4 +61,12 @@ const mapStateToProps = state => {
   }
 }
 
-export default connect(mapStateToProps)(Checkout)
+const mapDispatchToProps = dispatch => {
+  return {
+    checkoutCart: (userId = false) => {
```

Comment:
cartCheckout: (userId = false) => {
      dispatch(checkoutCart(userId))

Reply:
yes, will change them and commit again.

Reply:
also changed that (use cartCheckout) in the stripe button